.github/workflows/build.yml

```diff
@@ -0,0 +1,88 @@
+name: Build app package
+
+on: [ push, pull_request, workflow_dispatch ]
+
+jobs:
+  build_pkg:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Checkout Dependecies
+      run: |
+        git clone https://github.com/illusion0001/apollo-lib.git -b no-dbglgr --depth 1
+        git clone https://github.com/bucanero/zip.git                         --depth 1
+        git clone https://github.com/bucanero/oosdk_libraries.git             --depth 1
+        git clone https://github.com/bucanero/mxml                            --depth 1
+        git clone https://github.com/Al-Azif/ps4-skeleton.git                 --depth 1
+
+    - name: Set env vars
+      run: |
+        echo commit_ver=1.0`git rev-list HEAD --count`         >> $GITHUB_ENV
+        echo pkg_file=IV0000-APOL00004_00-APOLLO0000000PS4.pkg >> $GITHUB_ENV
+
+    - name: Download OpenOrbis Toolchain
+      run: |
+        curl -fLO https://github.com/illusion0001/OpenOrbis-PS4-Toolchain/releases/download/0.0.1.416/toolchain.tar.gz
+        tar xf toolchain.tar.gz
+        echo "OO_PS4_TOOLCHAIN=$GITHUB_WORKSPACE/OpenOrbis/PS4Toolchain" >> $GITHUB_ENV
+        cp build_rules.mk OpenOrbis/PS4Toolchain/build_rules.mk
+
+      # >10 is borked, for now
+    - name: Install LLVM and Clang 
+      uses: KyleMayes/install-llvm-action@v1
+      with:
+        version: "10.0"
+
+    - name: install zlib
+      run: |
+        cd oosdk_libraries/zlib_partial
+        make && make install
+
+    - name: install polarssl
+      run: |
+        cd oosdk_libraries/polarssl-1.3.9
+        make && make install
+
+    - name: install zip
+      run: |
+        cd zip
+        make && make install
+
+    - name: install apollo-lib
+      run: |
+        cd apollo-lib
+        make -f Makefile.PS4 && make -f Makefile.PS4 install
+
+    - name: install mini xml
+      run: |
+        cd mxml/ps4
+        make && make install
+
+    - name: install libjbc
+      run: cp -rf ps4-skeleton/libs/* $OO_PS4_TOOLCHAIN/lib
+
+    - name: copy SDL2 bin to SDK dir
+      run: wget https://cdn.discordapp.com/attachments/470415443115376643/959988630812762152/libSDL2.a -O $OO_PS4_TOOLCHAIN/lib/libSDL2.a
```

Comment:
Ideally this wouldn't be downloaded from Discord.

Reply:
SDL from pacbrew is incomplete, using prebuilt binary for now